docker/android.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Copyright (C) 2021 Helio Chissini de Castro <helio@kde.org>
```

Comment:
Change copyright to `Copyright (C) 2021 BMW CarIT GmbH` so its aligned with other files in this PR.